magento/pwa-studio
# PWA-2387: Replace Magento CE Homepage with Page Builder

- Deprecated CategoryList component
- Updated homepage to default to using pagebuilder
- Updated fallback messaging
- Updated test coverage



## Description
PWA Studio has two separate "homepage" contents: one for when Page Builder is present, and one when it isn't.

This was necessary because Page Builder used to be Adobe Commerce (formerly called "Enterprise Edition") only. Now that Page Builder is open source, the Magento Open Source (formerly called "Community Edition") version of our demo storefronts can confidently rely on Page Builder content being present.

## Related Issue







Closes [PWA-2387](https://jira.corp.magento.com/browse/PWA-2387)

## Acceptance

1. Venia homepage shows Page Builder content
2. The CategoryList component is marked as deprecated
3. When Page Builder CMS page content is not available, a fallback message appears instead

### Verification Stakeholders



### Specification




## Verification Steps





Preconditions:

1. Page Builder delivers CMS page content
2. PWA Studio is built / running with MAGENTO_BACKEND_EDITION=CE

Test Plan:

1. Deploy (or run locally) an instance of Venia with MAGENTO_BACKEND_EDITION=CE
2. Verify that the homepage shows PageBuilder CMS content
3. Verify the same when MAGENTO_BACKEND_EDITION=EE
4. Update backing instance Page Builder CMS content to not return anything (empty)
5. Verify that a fallback message is displayed instead


#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
